src/main/java/com/hemant/jlambda/App.java

```diff
@@ -30,6 +33,8 @@
 @Command(name = "jlambda")
 public class App implements Runnable {
 
+    private static final Logger logger = LogManager.getLogger(App.class);
+
```

Comment:
We should use sl4j APIs with a logback backend , so it would be easier to switch log implementations if needed